uptane-whitepaper.md

```diff
@@ -24,7 +24,7 @@ Yet, ignoring software updates is not a viable option either. As Uptane steering
+Increasingly, software updates for automotive ECUs are done using ["software-over-the-air”](https://ihsmarkit.com/research-analysis/remote-software-update-future-growth-business.html) (SOTA) strategies [8], in which revised versions of software programs are sent to vehicles over the Internet. This strategy is growing in popularity because it is a much quicker and more cost-effective delivery mechanism than traditional methods, such as distributing flash drives or requiring customers to bring in their vehicles for servicing. Furthermore, SOTA promises much higher penetration rates for essential updates. Established OEMs are recognizing the potential of this strategy. In May 2019, GM [announced](https://www.theverge.com/2019/5/21/18633000/gm-ota-software-updates-digital-platform-reuss) that the 2020 Cadillac CT5 sedan would feature an over-the-air software update mechanism, and that such systems will be standard on all GM vehicles in the next four years [9]. In the same year, Ford’s website announced that, beginning in 2020, it would begin equipping most redesigned vehicles in the U.S. with advanced over-the-air software update mechanisms, starting with the all-electric [Mustang Mach-E](https://media.ford.com/content/fordmedia/fna/us/en/news/2020/05/12/new-ford-over-the-air-updates-mustang-mach-e.html) [10]. Moreover, in 2021, [Toyota and Nissan](https://asia.nikkei.com/Business/Automobiles/Toyota-and-Nissan-to-upgrade-driving-functions-remotely) also begin rolling out OTA software update features, Toyota in the new Teammate driver assistance system for the Lexus LS, which will allow for hands-off driving on the highway, and Nissan in additions and adjustments to driving modes that improve the responsiveness of the steering wheel or reduce power consumption in the 2021 Ariya [11].  
```

Comment:
This hunk just removes some trailing whitespace and changes "for hands-off driving on the highway. and Nissan" -> "for hands-off driving on the highway, and Nissan"

Reply:
Thanks for taking such a close look. These changes are all fine.